pkg/adapter/v2/configurator_configmap.go

```diff
@@ -289,75 +207,11 @@ func (c *cloudEventsStatusReporterConfiguratorFromConfigMap) CreateCloudEventsSt
 		logger.Errorw("observability ConfigMap "+c.configMapName+" could not be retrieved", zap.Error(err))
 	}
 
-	var crStatusConfig map[string]string
-	if ocm != nil {
-		crStatusConfig = ocm.Data
-	}
-	r := crstatusevent.NewCRStatusEventClient(crStatusConfig)
+	cfg, err := o11yconfigmap.Parse(ocm)
```

Comment:
Looks like you need to check the `err` result here and do _something_ (I suspect)